07.bowling_object/lib/frame.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative './shot'
```

Comment:
```suggestion
require_relative 'shot'
```

require_relativeを使うのであればこちらがいいと思います〜